### ISSUE TITLE
A question about obb task

### ISSUE THREAD

**Issue by GodofCCode** [ultralytics/ultralytics#15805]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Recently i want to location this bottle which i want to obtain it's center and angle, but i only have one product. Then i triggrr it by different angle.  i label it like this
![val_batch2_labels](https://github.com/user-attachments/assets/db3b7355-3e82-4f6f-8065-d47505dc16a7)
then when i train it 100 epoch, i get the result like this
![val_batch1_pred](https://github.com/user-attachments/assets/f28aa236-291f-4de9-b252-27b4881354d2)
i don't know why. So i test it use different method.Finally i label it like this 
![val_batch0_labels](https://github.com/user-attachments/assets/85d4f223-1c88-4d86-831a-4a8e8f91706e)
i get the train result like this 
![val_batch0_pred](https://github.com/user-attachments/assets/d3466a10-c2bc-4535-b0c5-eebd26f0ba4a)
 i think when i label it the rectangle like square, i lost the angle information, but i label it when height obviously greater then width, it can correctly fitting. Idon't know why, so it's reason maybe overfitting or other?

**Comment by glenn-jocher** [comment]
@GodofCCode it looks like the issue might be related to how the labels are being interpreted by the model. Ensure that your labeling format is consistent and accurately reflects the orientation of the object. Also, try increasing the diversity of your training data to help the model generalize better. If the issue persists, please verify if it occurs with the latest version of the Ultralytics package.
